### DISCUSSION TITLE
Arrow key presses behavior

### DISCUSSION THREAD

**Discussion by YousefImran** [IntelliTect/EssentialCSharp.Web#245] [Feedback and Ideas] [answered]
I will prefix this by saying I don't know much about the tool yet, so ignorance is present.
That being said, when I am typing notes on a highlight and trying to use the arrow keys (mainly left and right to move between letters) it seems that these keys have predefined functionality which is move between the highlights themselves or maybe move back and forth between the pages I am currently on.

Is there a way to avoid this? Maybe a setting or something where the arrow keys are just working within their context, as in, if the courser is on text, move between the text rather than move between notes or pages.

This has been very frustrating unfortunately and I appreciate any feedback.

Thank you.

**Comment by BenjaminMichaelis** [comment, marked as answer]
> I will prefix this by saying I don't know much about the tool yet, so ignorance is present. That being said, when I am typing notes on a highlight and trying to use the arrow keys (mainly left and right to move between letters) it seems that these keys have predefined functionality which is move between the highlights themselves or maybe move back and forth between the pages I am currently on.
> 
> Is there a way to avoid this? Maybe a setting or something where the arrow keys are just working within their context, as in, if the courser is on text, move between the text rather than move between notes or pages.
> 
> This has been very frustrating unfortunately and I appreciate any feedback.
> 
> Thank you.

This is very helpful feedback, thank you! I have opened up #246 for discussion on this issue as I agree it is something that would be nice to be improved upon. Thank you for this perspective and helping improve this site for learning! 

If you have any other feedback/suggestions on this or other issues please don't hesitate to reach out.

**Comment by BenjaminMichaelis** [comment]
Hello @YousefImran, thank you for your feedback, we have pushed #247 to the production site if you want to try it out and see if that solves the issue you were facing or if there remains problems!

**Reply by YousefImran** [reply]
I believe it is resolved. As far as I can test. The only place where it may not work (for your information in case a similar case is to be submitted in the future) is if the site itself was designed to capture and respond to certain keys. For example, this website [Essential C#](https://essentialcsharp.com/home) is meant to give an ebook experience, so the right-arrow and left-arrow navigate through pages as you would in an e-pub reader, I don't think you would have any power over that, so just in case someone complains about this.
But for standard HTML/CSS/JS sites, and to my ability to test, it's good.
